app/services/imo_api_service.rb

```diff
@@ -52,11 +51,10 @@ def send_notification(patient, message)
     response = execute_post(url, body: request_body)
     result = process_response(response, url, "notification")
 
-    return if result.nil?
-
-    unless patient.imo_authorization.status == result.to_s
+    if patient.imo_authorization.status != result.to_s && result.in?([:not_subscribed, :no_imo_account])
```

Comment:
I'm no longer allowing this to update to "subscribed". My next ticket is the invitation callback, so it won't be necessary.

Reply:
Correct. It also means that we won't even attempt to send a notification via Imo unless we know they've already subscribed.